apps/web-client/vite.config.js

```diff
@@ -1,13 +1,56 @@
 import { sveltekit } from "@sveltejs/kit/vite";
 import path from "path";
+import { SvelteKitPWA } from "@vite-pwa/sveltekit";
 
 import RollupNodePolyfill from "rollup-plugin-node-polyfills";
 
 const rushDir = path.join(__dirname, "..", "..", "common");
 
+export const DEFAULT_BASE_PATH = "/preview";
+
+const dev = process.env.NODE_ENV === "development";
+const CURRENT_SHA = process.env.CURRENT_SHA;
+let BASE_PATH = process.env.BASE_PATH;
+
+if (typeof BASE_PATH === "undefined") {
+	// If no BASE_PATH was passed as environment variable, we default to either
+	// `/preview` or (if CURRENT_SHA is defined) to `/preview/<CURRENT_SHA>`
+	BASE_PATH = dev || !CURRENT_SHA ? DEFAULT_BASE_PATH : `${DEFAULT_BASE_PATH}/${CURRENT_SHA}`;
+	process.env.BASE_PATH = BASE_PATH;
```

Comment:
Be helpful to leave a small comment here explaining why we are doing this: so that the default base path will be communicated to the `svelte.config`